thrift/src/main/thrift/atoms/media.thrift

```diff
@@ -72,11 +72,18 @@ struct PlutoData {
   3: optional string masterId
 }
 
+struct IconikData {
+  1: optional string workingGroupId
```

Comment:
these four fields comprise the core data that we need to identify an Iconik project